vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/PageSteps.java

```diff
@@ -248,7 +248,7 @@ public void isPageScrolledToAnElement(Locator locator)
      * <p>
      * @param relativeURL A string value of the relative URL
      */
```

Comment:
please don't forget to sync javadoc with main docs

Reply:
updated